src/applications/disability-benefits/all-claims/pages/pastEmploymentFormIntro.js

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import { validateBooleanGroup } from 'us-forms-system/lib/js/validation';
+import { unemployabilityTitle } from '../content/unemployabilityFormIntro';
+
+const pastEmploymentHistory = (
```

Comment:
this should be in a file in `src/applications/disability-benefits/all-claims/content/`

Reply:
moved!